test/e2e/apimachinery/chunking.go

```diff
@@ -30,12 +30,18 @@ import (
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apiserver/pkg/features"
 	"k8s.io/apiserver/pkg/storage/storagebackend"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	"k8s.io/client-go/util/workqueue"
 	"k8s.io/kubernetes/test/e2e/framework"
 	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
 )
 
+func shouldCheckRemainingItem() bool {
+	return utilfeature.DefaultFeatureGate.Enabled(features.RemainingItemCount)
+}
```

Comment:
This isn’t set magically by e2e reading from the server, right?

Reply:
No, it's local.

Reply:
Ok, and these aren’t conformance tests yet?

Reply:
No. Conformance tests starts with "framework.ConformanceIt".